website/aws.erb

```diff
@@ -249,6 +249,9 @@
                         </li>
                         <li<%= sidebar_current("docs-aws-datasource-network-interface") %>>
```

Comment:
Minor nitpick: We don't expect most people to know this, but when we add a plural data source, we need to adjust the `sidebar_current` of the singular data source in the sidebar page and the top matter of its documentation page. Otherwise the sidebar will highlight both the singular and plural data sources when visiting the singular page since its sidebar_current becomes a prefix of the plural (unless its adjusted).

e.g. Changing here and `website/docs/d/network_interface.html.markdown` is the trick we use:

```
sidebar_current("docs-aws-datasource-network-interface-x")
```